.ci.yaml

```diff
@@ -60,8 +60,7 @@ targets:
       - .ci.yaml
 
   - name: Mac codesign
-    bringup: true
-    postsubmit: false
+    presubmit: false
```

Comment:
Should this run on presubmit?

Reply:
umm my understanding is that like Casey said, presubmit would use a try builder to write to cipd, so we might want to avoid this and not building and publishing on presubmit

Reply:
The recipe should be updated to only upload in prod

Reply:
umm device doctor and codesign shares the same recipe. Like Casey pointed out, device doctor needs the presubmit try builders to upload the cipd packages. If updating to only upload in prod it would also affect device doctor. umm I probably don't have access, shall we modify ACL for device doctor?

Reply:
I sent https://flutter-review.googlesource.com/c/recipes/+/34580

Reply:
Recipes is better for things that will have ACL issues (like whether to upload artifacts), and ci.yaml is better for build/test configurations. In this case, you can configure in ci.yaml what cipd packages should be built, but recipes should have the upload logic.

Reply:
Thanks for explaining Casey!